praw/exceptions.py

```diff
@@ -17,22 +17,58 @@ class PRAWException(Exception):
 class APIException(PRAWException):
     """Indicate exception that involve responses from Reddit's API."""
 
-    def __init__(self, error_type: str, message: str, field: Optional[str]):
+    @staticmethod
```

Comment:
I believe as I mentioned elsewhere, I am not a fan of having a list of errors on APIException. That would mean every subclass of APIException would presumably also have a list, which is incorrect. Can we make a single class, which is a subclass of APIException that holds such a list? MultiException or something similar.

Reply:
So would it be better to have an exception that's just a list of APIException's?

Reply:
Yes.